data/membership-schema/src/main/resources/net/corda/schema/membership/mgm/registration/1.0/corda.mgm.registration.json

```diff
@@ -79,6 +79,14 @@
         "1.3"
       ]
     },
+    "corda.group.tls.type": {
+      "description": "The TLS type. Default to OneWay",
```

Comment:
We could add more detail here. e.g. copy the docs from the gateway json schema.
```
          "description": "What kind of TLS connections to establish between the gateways in our cluster and gateways in other clusters. This setting MUST be the same on all Gateways that need to communicate via HTTPS.",
          "oneOf": [
            {"const": "ONE_WAY", "title": "Establishes a regular TLS connection, where the server gateway will be authenticated."},
            {"const": "MUTUAL", "title": "Establishes a mutual TLS connection, where both the server and client gateway are authenticated."}
```

Reply:
I want to make it case insensitive, so I prefer not to put `oneOf` (as the rest of the doc).

Reply:
I meant reformat it (I was being lazy and just copied the whole thing).

Reply:
What you did looks great.